types.go

```diff
@@ -235,6 +236,14 @@ func (p GetParams) Format() string {
 	return p.RawFormat
 }
 
+func (p GetParams) Download() bool {
+	if !p.SkipDownload {
+		return true
+	}
+
+	return !p.SkipDownload
+}
+
```

Comment:
I don't think it's necessary to create a method for this - looking at how it gets called, it should be enough to reference the `SkipDownload` struct field directly. It's an understandable instinct to create seam for this functionality in case further business logic gets added later, but let's favour simplicity for the time being.